features/steps/utils.py

```diff
@@ -139,6 +141,30 @@ def _clean(loggable):
     return data
 
 
+def traverse(resource, path):
```

Comment:
I'm late to the party, and this is already merged, but is there a reason you didn't import `traverse` from argonaut.py?

Reply:
Thanks for the review @erikwiffin! This was a net move of the `traverse` code, from Argonaut to utils. I think the idea is it's useful for "more than just Argonaut".

Reply:
Ah, thanks! That makes sense.